README.md

```diff
@@ -20,3 +20,7 @@ There is currently no official installer for Earth Enterprise Fusion & Server.
 
 ### Known Issues
 Known issues can be tracked via [wiki page](https://github.com/google/earthenterprise/wiki/Known-Issues), which is updated weekly.
+
+### Slack
+<img src="https://github.com/google/earthenterprise/blob/master/slack_rgb.png" width="100"> Join the OpenGEE [Slack channel](http://slack.opengee.org)! 
```

Comment:
I know this isn't merged yet, but I don't believe that it will end up being the correct URL. For instance, here is that URL to your repo: https://github.com/tst-ahernandez/earthenterprise/blob/slack_badge/slack_rgb.png

It takes you to a webpage instead of the image itself. I think we might be better off adding it to the wiki and referencing it that way.

<img src="https://github.com/tst-ahernandez/earthenterprise/blob/slack_badge/slack_rgb.png">